docs/aws.md

```diff
@@ -0,0 +1,97 @@
+# Deploying to AWS on Amazon Linux 2
+
+This guide describes how to launch an AWS instance running Amazon Linux 2 and configure it to run Swift. The approach taken here is a step by step approach through the console. This is a great way to learn, but for a more mature approach we recommend using Infrastructure as Code tools such as AWS Cloudformation, and the instances are created and managed through automated tools such as Autoscaling Groups. For one approach using those tools see this blog article: https://aws.amazon.com/blogs/opensource/continuous-delivery-with-server-side-swift-on-amazon-linux-2/
+
+## Launch AWS Instance
+
+Use the Service menu to select the EC2 service.
+
+![Select EC2 service](../images/aws/services.png)
+
+Click on "Instances" in the "Instances" menu
+
+![Select Instances](../images/aws/ec2.png)
+
+Click on "Launch Instance", either on the top of the screen, or if this is the first instance you have created in the region, in the main section of the screen.
+
+![Launch instance](../images/aws/launch-0.png)
+
+Choose an Amazon Machine Image (AMI). In this case the guide is assuming that we will be using Amazon Linux 2, so select that AMI type.
+
+![Choose AMI](../images/aws/launch-1.png)
+
+Choose an instance type. Larger instances types will have more memory and CPU, but will be more expensive. To experiment is it frugal to choose a smaller instance type. In this case I have a `t2.micro` instance type selected.
```

Comment:
`frugal` seems like a very Amazon word. I know what you mean but it might be less obvious for someone else. Maybe something like-

```suggestion
Choose an instance type. Larger instances types will have more memory and CPU, but will be more expensive. A smaller instance type will be sufficient to experiment. In this case I have a `t2.micro` instance type selected.
```